lib/play-audio.coffee

```diff
@@ -2,7 +2,10 @@ path = require "path"
 
 module.exports =
   play: ->
-    pathtoaudio = path.join(__dirname, '../audioclips/gun.wav')
+    if (@getConfig "audioclip") is "customAudioclip"
+      pathtoaudio = path.join(__dirname,"../audioclips/" + @getConfig "customAudioclip")
+    else
+      pathtoaudio = path.join(__dirname, @getConfig "audioclip");
     audio = new Audio(pathtoaudio);
```

Comment:
A little bit paranoid here, what happens when the audio is not found? 
Maybe a try/catch or a check with a fallback to the default audio clip.